Dockerfile-flask

```diff
@@ -1,4 +1,5 @@
 FROM python:3.10-slim
+ARG POETRY_VERSION=1.2.0
```

Comment:
Why 1.2.0 ?
https://pypi.org/project/poetry/#history

Reply:
Right, I will update it to the latest version.
Probably copy/paste mistake.